gnome-initial-setup/eos-demo-mode-force-gdm-restart

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+systemctl reload gdm.service
```

Comment:
Hmm, it may have been on the session bus. I guess `g_application_get_dbus_connection` returns the session bus connection...